### ISSUE TITLE
"layoutInvalidationKey" is not forcing remeasure if the props.data is not changed.

### ISSUE THREAD

**Issue by vinicz** [computerjazz/react-native-draggable-flatlist#168]
**Describe the bug**
`layoutInvalidationKey` is not forcing remeasure if the props.data is not changed. This could happen if some other property is changed together with the `layoutInvalidationKey` without data change. In our case, we wanted to update the padding based on the size of another component.
A video about the issue:
![Hove element issue](https://i.imgur.com/LEOWZi0.gif)

**To Reproduce**
To reproduce the issue a property like the padding of the container needs to be changed after all the measurements were done. An example of this:
```Javascript
import React, {Component} from 'react';
import {View, TouchableOpacity, Text} from 'react-native';
import DraggableFlatList, {
  RenderItemParams,
} from 'react-native-draggable-flatlist';

const exampleData = [...Array(20)].map((d, index) => ({
  key: `item-${index}`, // For example only -- don't use index as your key!
  label: index,
  backgroundColor: `rgb(${Math.floor(Math.random() * 255)}, ${
    index * 5
  }, ${132})`,
}));

class Example extends Component {
  state = {
    data: exampleData,
    padding: 0,
  };

  componentDidMount(): void {
    setTimeout(() => this.setState({padding: 200}), 2000);
  }

  renderItem = ({item, drag, isActive}: RenderItemParams<any>) => {
    return (
      <TouchableOpacity
        style={{
          height: 100,
          backgroundColor: isActive ? 'blue' : item.backgroundColor,
          alignItems: 'center',
          justifyContent: 'center',
        }}
        onLongPress={drag}>
        <Text
          style={{
            fontWeight: 'bold',
            color: 'white',
            fontSize: 32,
          }}>
          {item.label}
        </Text>
      </TouchableOpacity>
    );
  };

  render() {
    return (
      <View style={{flex: 1}}>
        <DraggableFlatList
          data={this.state.data}
          renderItem={this.renderItem}
          keyExtractor={(item) => `draggable-item-${item.key}`}
          onDragEnd={({data}) => this.setState({data})}
          layoutInvalidationKey={this.state.padding.toString()}
          contentContainerStyle={{
            paddingTop: this.state.padding,
          }}
        />
      </View>
    );
  }
}

export default Example;
```

**Platform & Dependencies**
 - Platform: iOS and Android
 - React Native version: 0.62.1
 - Reanimated version: 1.7.1
 - React Native Gesture Handler version: 1.6.1

**Additional context**
A possible fix would be the add an extra condition when triggering the measuring process, like this: https://github.com/vinicz/react-native-draggable-flatlist/commit/df448f947b83d9eab188cc21447aa182365fc30a#diff-38682f47b0cf12c516bc3714fc45d0fdR266

I could open a PR with the change above if you don't see any issue with it.

**Comment by MattFroggett** [comment]
Hey @computerjazz it looks like you fixed this with 5f8cd118f4743a661b4f1c32ffd12faeae07b190, would it be possible to publish a new version to npm please?

**Comment by computerjazz** [comment]
published in `2.3.3`
